src/snackbar/snackbar.ios.ts

```diff
@@ -132,7 +159,7 @@ export class SnackBar extends SnackBarBase {
             this._snackbarManager.delegate = null;
             resolve({
                 action: SnackBarAction.DISMISS,
-                reason: userInitiated ? DismissReasons.ACTION : DismissReasons.TIMEOUT
+                reason: userInitiated ? DismissReasons.ACTION : DismissReasons.TIMEOUT,
```

Comment:
the reason for having `,` at the end of multi-line objects it's to avoid merge conflicts.

https://medium.com/@nikgraf/why-you-should-enforce-dangling-commas-for-multiline-statements-d034c98e36f8

Reply:
@henrychavez nice! But not all ts files have been changed.
Let's make it in another work load. It makes the PR hard to review.
But i like the idea and will implement the change